docs/how-to/integrate_oai_ran_with_ue_simulator.md

```diff
@@ -0,0 +1,97 @@
+# Integrate Charmed OAI RAN with the OAI RAN UE Simulator
+
+Charmed OAI RAN includes a UE simulator that can be used to simulate 5G network traffic and validate the RAN deployment.
+
+This guide walks you through deploying the Charmed OAI RAN UE Simulator on a Kubernetes cluster using Juju and Terraform.
+
+## 1. Configure Charmed OAI RAN DU to run in simulation mode
+
+Charmed OAI RAN DU needs to be configured in simulation mode to work with the UE simulator.
+
+To enable the DU to run in simulation mode, set the `simulation-mode` configuration option to `true`:
+
+```console
+juju config du simulation-mode=true
+```
```

Comment:
Depending on how the module is structured, this might not work. What I mean is that if the `ue.tf` file is put in the same directory as the `ran.tf` file, when running `terraform apply`, that setting will be overwritten. I would suggest doing it all through Terraform.

Reply:
Agreed, done